### ISSUE TITLE
Installation is not idempotent

### ISSUE THREAD

**Issue by dzubchik** [roquie/ansible-kontena-server#3]
Each run creates new vault keys. I propose to add option to regenerate them, but remove this as default behavior.

**Comment by roquie** [comment]
Okay. I think It to be possible if add the following condition (2 times):
```
when: ks_vault_key|default('')|length == 0 and ks_vault_generate
```

**Comment by roquie** [comment]
It's normal solution for you?

**Comment by dzubchik** [comment]
Looks good to me. And would be nice to have the test for this, because it's very dangerous option.

**Comment by roquie** [comment]
look this pls https://github.com/roquie/ansible-kontena-server/pull/4

**Comment by dzubchik** [comment]
Thanks, I will look closely this PR before Monday.

**Comment by dzubchik** [comment]
Yes, it solves current issue. During testing I realized that here are a larger issue with installation process. I will send PR for it.
